doc/optimizers.md

```diff
@@ -518,6 +517,7 @@ Differential Evolution is an evolutionary optimzation algorithm which selects be
 * `DE(`_`populationSize, maxGenerations`_`)`
 * `DE(`_`populationSize, maxGenerations, crossoverRate`_`)`
 * `DE(`_`populationSize, maxGenerations, crossoverRate, differentialWeight`_`)`
+* `DE(`_`populationSize, maxGenerations, mutationProb, mutationSize, tolerance`_`)`
```

Comment:
Should this be `populationSize, maxGenerations, crossoverRate, differentialWeight, tolerance`?